test/evergreen.yml

```diff
@@ -923,6 +923,19 @@ tasks:
 
   # End of Python unit test tasks
 
+  - name: s-all
+    depends_on:
+    - name: compile
```

Comment:
Some special configure option, preloaded library and env variables currently set in `wiredtiger-pull-request-linux` job (pasted below) are not covered by the `compile` task. 
```../configure CFLAGS="-ggdb" --enable-silent-rules --enable-strict --enable-diagnostic --disable-static ; make -j2
[ -e /usr/local/lib/libeatmydata.so ] && export LD_PRELOAD=/usr/local/lib/libeatmydata.so
export TESTUTIL_DISABLE_LONG_TESTS=1
```